demo/dlb-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+enable_and_configure_vfs() {
+  DEVPATH=$1
+  NUM_VFS_PER_PF=$2
+  # enable sriov
+  SRIOV_NUMVFS_PATH="$DEVPATH"/sriov_numvfs
+  if test -w "$SRIOV_NUMVFS_PATH" -a "$(cat "$SRIOV_NUMVFS_PATH")" -eq 0 ; then  
```

Comment:
If ```test -w``` fails I'd consider this as an error, which is worth reporting instead of silently skipping configuration.
Already configured devices (with non-zero sriov_numvfs) should be also reported.

Reply:
AHAA!!!! It was the reason why it fails..!!
I see.!
Let me then,, divide the if statement into two layered ifs